### PR TITLE
#735: tests: COMMAND GETKEYS, COMMAND INFO, COMMAND LIST, COMMAND RENAME, COPY

### PR DESCRIPTION
**Pull request by Indranil0603**
for #735 

Adds support for subcommands using HTTP, and integrations tests for HTTP support of the following commands

Changes:

- redisCmdAdapter.go - added support for subcommands
- command_getkeys_test.go - Integration test for COMMAND GETKEYS
- command_info_test.go - Integration test for COMMAND INFO
- command_list_test.go - Integration test for COMMAND LIST
- command_rename_test.go - Integration test for COMMAND RENAME
- copy_test.go - Integration test for COPY

### CODE REVIEW

File: internal/server/utils/redisCmdAdapter.go [hunk 1]
```diff
@@ -28,14 +28,29 @@ const (
...
+	}
+
 	var args []string
 
+	//Handle subcommand and multiple arguments
+	if subcommand != "" {
+		args = append(args, subcommand)
+
+		if len(commandParts) > 2 {
+			args = append(args, commandParts[2:]...)
```
Comment (pratikpandey21) [inline internal/server/utils/redisCmdAdapter.go:50]: Are there any cases for this scenario?

File: integration_tests/commands/http/command_getkeys_test.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package http
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetKeys(t *testing.T) {
```
Comment (pratikpandey21) [inline integration_tests/commands/http/command_getkeys_test.go:9]: Please use the same set of tests that exists for async

**Review by pratikpandey21 — COMMENTED** [review]
Looks good. Left a few comments

File: integration_tests/commands/http/command_getkeys_test.go (hunk 2 above)
Comment (Indranil0603) [reply integration_tests/commands/http/command_getkeys_test.go]: Yes, I have added  the same set of tests that exists for async

**Comment by JyotinderSingh** [comment]
Please resolve conflicts, will merge once done.

**Comment by Indranil0603** [comment]
@JyotinderSingh i have resolved the conflicts

**Pull request merged**